src/sonic-config-engine/tests/multi_npu_data/sample-minigraph.xml

```diff
@@ -1115,6 +1143,141 @@
           <Speed>40000</Speed>
         </a:EthernetInterface>
       </EthernetInterfaces>
+      <SystemPorts>
```

Comment:
I think the system ports needs to be defined inside each ASIC, not at the line card level.

Reply:
Each asic needs to know about all the system ports in the system. Currently system ports are initialized during orchagent init so this makes it easier to have them defined in `DeviceInfo` and parsed by each asic (either in multi-asic setup or in single-asic linecards) than having them duplicated for each asic in the minigraph.xml